src/prost/cosmos.auth.v1beta1.rs

```diff
@@ -94,1785 +94,3 @@ impl ::prost::Name for Params {
         "/cosmos.auth.v1beta1.Params".into()
```

Comment:
We still have some remaining cosmos built files. Is this alright?

Reply:
Not sure why this is being generated, but it's not being compiled nor exported at least.

Reply:
Ah. They are generated because there are some dependencies to them in some IBC apps. I think we can remove them - the same way you `rm` some redundant generated files here.

https://github.com/cosmos/ibc-proto-rs/blob/47f4e508162c0bfbee454718e2668e19c74016ff/scripts/sync-protobuf.sh#L210-L219

Reply:
Done